src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -178,8 +178,6 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*InferRequestIOBBlobTest.*canProcessDeallocatedOutputBlobAfterGetAndSetBlob.*)",
         // Plugin version was changed to ov::Version
         R"(.*VersionTest.*pluginCurrentVersionIsCorrect.*)",
-        // Issue: 120286
-        R"(.*smoke_Basic/FuseTransposeAndReorderTest.CompareWithRefs.*)",
```

Comment:
could you please clarify why `toLegacyType` affects CPU tests?
From my point of view, CPU does not use `src/inference/src/cnn_network_ngraph_impl.cpp` anymore and any changes there don't affect CPU tests.

Moreover, adding new conversions to `toLegacyType` can break API 1.0 behavior

Reply:
The issue is from ticket: https://jira.devtools.intel.com/browse/CVS-120286, I have some analysis in this ticket.
It is legacy API test case, which causes transpose node is not fused as expected due to additional convert node is added before result node.
I'm not sure whether we need this PR considered legacy API will be deprecated in 24.0.

Reply:
Maybe we need to rewrite this test to new API then?

Reply:
PR migrate FuseTransposeAndReorderTest to 2.0: https://github.com/openvinotoolkit/openvino/pull/21297